spec/setup.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Setup file for running tests.
+ */
+
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const { EOL } = require('os');
+const { exec } = require('child_process');
+
+const WHITELISTED_ACCOUNTS = 'WHITELISTED_ACCOUNTS';
+const CLIENT_SECRET = 'CLIENT_SECRET';
+const NEW_RELIC = 'NEW_RELIC_NO_CONFIG_FILE';
+const NEW_RELIC_APP = 'NEW_RELIC_APP_NAME';
+const envPath = path.join(__dirname, '..', '.env');
+const envExamplePath = path.join(__dirname, '..', '.env.example');
+let envData = '';
+
+const setWhitelistedAccount = () => {
+  // Load env.
+  let data = '';
+  if (fs.existsSync(envPath)) {
+    data = fs.readFileSync(envPath);
+  } else {
+    data = fs.readFileSync(envExamplePath);
```

Comment:
Have a check here: if example file is missing as well, throw an error asking the user to update their branch.

Reply:
Done.